clients/js/test/compat.spec.ts

```diff
@@ -430,39 +301,39 @@ describe('window.ethereum', () => {
   });
 });
 
-describe('legacy MetaMask', () => {
-  it('proxy', async () => {
-    const wrapped = wrap(new MockLegacyProvider(), cipher);
-    expect(wrapped.isMetaMask).toBe(false);
-    expect(wrapped.isConnected()).toBe(false);
-    expect((wrapped as any).sapphire).toMatchObject({ cipher });
-  });
-
-  runTestBattery(async () => {
-    const provider = new MockLegacyProvider();
-    const wrapped = wrap(provider, cipher);
-    const signer = new ethers5.providers.Web3Provider(wrapped).getSigner();
-    const rawSign: ethers5.Signer['signTransaction'] = async (
-      ...args: unknown[]
-    ) => {
-      return new Promise((resolve, reject) => {
-        wrapped.send(
-          {
-            method: 'eth_signTransaction',
-            params: args,
-          },
-          (err, ok) => {
-            if (err) reject(err);
-            else resolve((<any>ok).result as string);
-          },
-        );
-      });
-    };
-    return [signer, provider, rawSign];
-  });
-});
-
-function runTestBattery<S extends ethers5.Signer | ethers6.Signer>(
+// describe('legacy MetaMask', () => {
```

Comment:
is this coming back or will it go away?

Reply:
Was on the fence about removing these as other wrappers have been conservative, but will poll the community. I'm trigger happy here in general.